be2-scala/src/main/scala/ch/epfl/pop/model/objects/ElectionChannel.scala

```diff
@@ -0,0 +1,64 @@
+package ch.epfl.pop.model.objects
+
+import akka.pattern.AskableActorRef
+import ch.epfl.pop.model.network.method.message.Message
+import ch.epfl.pop.model.network.method.message.data.election.{CastVoteElection, SetupElection}
+import ch.epfl.pop.pubsub.AskPatternConstants
+import ch.epfl.pop.storage.DbActor
+import ch.epfl.pop.storage.DbActor.{DbActorReadAck, DbActorReadChannelDataAck}
+
+import scala.collection.mutable
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Await, Future}
+
+object ElectionChannel {
+  implicit class ElectionChannel(channel: Channel) extends AskPatternConstants {
```

Comment:
```suggestion
  implicit class ElectionChannelExtensionMethods(channel: Channel) extends AskPatternConstants {
```
Since this is not really a channel, it might be better to give it this kind of naming convention, in line with https://www.scala-lang.org/api/2.13.8/scala/collection/IterableOnceExtensionMethods.html. That should prevent confusion.